optuna/visualization/matplotlib/_optimization_history.py

```diff
@@ -72,7 +68,7 @@ def _get_optimization_history_plot(
 ) -> "Axes":
     # Set up the graph style.
     plt.style.use("ggplot")  # Use ggplot style sheet for similar outputs to plotly.
-    _, ax = plt.subplots()
+    _, ax = plt.subplots(tight_layout=True)
```

Comment:
Only providing `tight_layout=True` is not sufficient for Ubuntu.
You also need to calibrate the legend location to look prettier.

<img width="640" height="480" alt="this_pr" src="https://github.com/user-attachments/assets/cc7df6f0-2605-451e-be92-6a45247b65cc" />


For example, see below:

<img width="640" height="480" alt="example" src="https://github.com/user-attachments/assets/c39373c4-7373-47c9-bdbe-419dee4ad140" />

Reply:
I think it’s fine since the output of the figure looks the same as the Plotly version.

<img width="642" height="468" alt="newplot" src="https://github.com/user-attachments/assets/d4c22153-a253-4c2f-97af-ba748b68e90a" />

Reply:
That's right. (IMO, the plotly version should be modified though)